problem_builder/answer.py

```diff
@@ -53,6 +52,11 @@ class AnswerMixin(XBlockWithPreviewMixin, XBlockWithTranslationServiceMixin, Stu
     """
     Mixin to give an XBlock the ability to read/write data to the Answers DB table.
     """
+    USER_STATE_FIELDS = ['edxnotes_visibility', ]
+    USER_STATE_TRANSFORMS = {
+        'answer_data': lambda self: self.get_answer_data()
```

Comment:
Could we change this API to remove the need for a lambda function - so we can just write `'answer_data': self.get_answer_data` ?

Reply:
Also, we don't return `created_on` nor `modified_on` for any other component types (MCQ, MRQ, etc.) so I think we should just get rid of `get_answer_data` and instead just put

```python
USER_STATE_TRANSFORMS = {
    'student_input': self.student_input
}
```